src/map/utils/battleutils.cpp

```diff
@@ -1723,6 +1723,15 @@ namespace battleutils
                     parryRate = parryRate + issekiganBonus;
                 }
 
+                // Inquartata grants a flat parry rate bonus.
+                if (CCharEntity* PChar = dynamic_cast<CCharEntity*>(PDefender))
```

Comment:
Sverdheid and Zurko-Bazurko are RUN mobs that players need to fight.

Reply:
I think ibm's idea was that if you aren't needing the trait check, then there's no need to cast to CCharEntity*. You can just use PDefender directly as CBattleEntity has the getMod method. :)